### PR TITLE
Grzegorz favourite screen

### PR DESCRIPTION
**Pull request by GrzegorzKwasniewski**

### CODE REVIEW

File: app/src/main/java/com/example/android/psiak/ui/favouriteView/TabsPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import com.example.android.psiak.R;
+
+public class TabsPagerAdapter extends FragmentPagerAdapter {
+
+    // region Properties
+
+    private int TABS_ITEMS_COUNT = 4;
```
Comment (mbialorucki) [inline app/src/main/java/com/example/android/psiak/ui/favouriteView/TabsPagerAdapter.java:15]: static final?

**Review by PawelZarzycki — APPROVED** [review]
Ok

File: app/src/main/java/com/example/android/psiak/data/network/Repository.java [hunk 2]
```diff
@@ -68,6 +70,14 @@
         ArrayList<DogFirebase> getCachedDogs();
```
Comment (mbialorucki) [inline app/src/main/java/com/example/android/psiak/data/network/Repository.java:70]: You can use List here - less coupled code

File: app/src/test/java/com/example/android/psiak/FirebasePresenterTest.java [hunk 3]
```diff
@@ -21,13 +23,23 @@
 
     private AddAnimalContract.View mockView;
     private MockFirebaseRepository mockFirebaseRepository;
+<<<<<<< HEAD
```
Comment (mbialorucki) [inline app/src/test/java/com/example/android/psiak/FirebasePresenterTest.java:26]: Unresolved merge conflict?

**Review by mbialorucki — APPROVED** [review]

**Pull request merged**